gsuite_reports_rules/gsuite_permissions_delegated.py

```diff
@@ -11,5 +14,13 @@ def rule(event):
 
 
 def title(event):
-    return 'User [{}] was delegated new administrator privileges'.format(
-        event.get('actor', {}).get('email'))
+    role = '<UNKNOWN_ROLE>'
+    user = '<UKNNOWN_USER>'
+    for details in event.get('events', [{}]):
+        if (details.get('type') == 'DELEGATED_ADMIN_SETTINGS' and
+                details.get('name') == 'ASSIGN_ROLE'):
```

Comment:
Makes me a bit sad that we have to duplicate the rule logic exactly to accomplish this :/ These g suite logs really are ugly.